guides/v2.3/inventory/source-selection-algorithms.md

```diff
@@ -0,0 +1,167 @@
+---
+group: inventory
+title: Source selection algorithms
+---
+
+The **Source Selection Algorithm (SSA)** receives the total number of shippable SKUs and their quantities in an order as well as the order's shipping address. It recommends how to fulfill partial and full shipments. The merchant decides which business needs take precedence when deciding which shipping method to use:
+
+* Should the products be delivered from the sources designated as having the highest priority? 
+* Should the total shipment cost be the primary factor in choosing a shipment method?
+* Should the shipments originate from the closest source?
+
+Magento 2.3 provides the priority-based SSA only. Third party developers can create their own algorithms to 
+help merchants decide which shipping option best meets their needs.
```

Comment:
Might be worth noting they can develop SSAs to run during shipment creation or during checkout. Would also note in the first sentence the priority-based SSA runs during shipment creation.